DevEdu.API/DevEdu.API/Configuration/MapperProfile.cs

```diff
@@ -44,8 +45,7 @@ private void CreateMappingToDto()
             CreateMap<LessonInputModel, LessonDto>()
                 .ForMember(dest => dest.Teacher, opt => opt.MapFrom(src => new UserDto { Id = src.TeacherId }))
                 .ForMember(dest => dest.Date, opt => opt.MapFrom(src => DateTime.ParseExact(src.Date, _dateFormat, CultureInfo.InvariantCulture)));
-            CreateMap<LessonUpdateInputModel, LessonDto>()
-                .ForMember(dest => dest.Date, opt => opt.MapFrom(src => DateTime.ParseExact(src.Date, _dateFormat, CultureInfo.InvariantCulture)));
```

Comment:
why have you removed this?

Reply:
I deleted the date because it didn't change in the procedure, but now I think it's wrong. We need to add a date change.